tests/Unit/Domain/CoordinateMaps/Test_Frustum.cpp

```diff
@@ -302,6 +302,196 @@ void test_bulged_frustum_inv_map() {
   test_inverse_map(map, test_point3);
   test_inverse_map(map, test_point4);
 }
+
+void test_bulged_frustum_equiangular_full() {
+  INFO("Bulged frustum jacobian");
+  const std::array<std::array<double, 2>, 4> face_vertices{
+      {{{-2.0, -2.0}}, {{2.0, 2.0}}, {{-4.0, -4.0}}, {{4.0, 4.0}}}};
+  const CoordinateMaps::Frustum map(
+      face_vertices, 2.0, 4.0, OrientationMap<3>{}, true, 1.0, false, 1.0, 0.0);
+
+  // Points on the upper +zeta face:
+  const std::array<double, 3> lowerxilowereta{{-1.0, -1.0, 1.0}};
+  const std::array<double, 3> zeroxilowereta{{0.0, -1.0, 1.0}};
+  const std::array<double, 3> upperxilowereta{{1.0, -1.0, 1.0}};
+
+  const std::array<double, 3> lowerxizeroeta{{-1.0, 0.0, 1.0}};
+  const std::array<double, 3> zeroxizeroeta{{0.0, 0.0, 1.0}};
+  const std::array<double, 3> upperxizeroeta{{1.0, 0.0, 1.0}};
+
+  const std::array<double, 3> lowerxiuppereta{{-1.0, 1.0, 1.0}};
+  const std::array<double, 3> zeroxiuppereta{{0.0, 1.0, 1.0}};
+  const std::array<double, 3> upperxiuppereta{{1.0, 1.0, 1.0}};
+
+  // If Frustum is successfully bulged, points on upper +zeta face
+  // should have the same radius:
+  CHECK(magnitude(map(lowerxilowereta)) == approx(4.0 * sqrt(3.0)));
```

Comment:
Perhaps put the points defined above in a std::vector so you can easily check them all?

Also, for readability it would be useful to define `radius = 4 * sqrt(3)` and also use it in the tests below.